pages/tech/contributing.md

```diff
@@ -0,0 +1,87 @@
+import Section from '../../components/section'
+
+# Contribution Guide
+
+This page serves as a general contribution guide for CarbonPlan data science projects. Some projects within the organization may also provide customized contribution guides.
+
+## Getting Started
+
+If you're new to contributing to CarbonPlan projects, here are some resources to help you get started:
+
+- Familiarize yourself with our [GitHub organization](https://github.com/carbonplan)
+- Read our [Code of Conduct](./conduct)
```

Comment:
I worry a little about this separate copy becoming out-of-date with the [version on GitHub](https://github.com/carbonplan/carbonplan.org?tab=coc-ov-file). What do you think about removing that page and just referencing the GitHub version directly?

Reply:
good point. i had forgotten about the GitHub version. I will remove this and link to https://github.com/carbonplan/.github/blob/main/CODE_OF_CONDUCT.md